src/vector/feature.rs

```diff
@@ -87,4 +150,12 @@ impl FieldValue {
             _ => panic!("not a RealValue")
         }
     }
+
+    /// Interpret the value as `i32`. Panics if the value is something else.
+    pub fn as_int(self) -> i32 {
```

Comment:
Methods that consume `self` are usually named `into_<type>`, so in this case, it would be `into_int`, but I notice above that the other methods on this structure use `as_<type>`.